src/utils/codemods/resolveCodemod.js

```diff
@@ -0,0 +1,9 @@
+const path = require('path')
+
+module.exports.resolveCodemod = pathSegments => path.join(
+    __dirname,
+    '..',
+    '..',
+    'codemods',
+    ...pathSegments.split('/')
```

Comment:
➕ 1 - we also use this pattern (albeit a bit more dynamic, since we accept `--cwd` arg on the commandline) in the [platform cli](https://github.com/dhis2/app-platform/blob/master/cli/src/lib/paths.js)